api/src/controllers/put/collection-bcmi.js

```diff
@@ -0,0 +1,95 @@
+const mongoose = require('mongoose');
```

Comment:
Same as the comment for post/collection-bcmi.... role check for admin:bcmi explicity?

Reply:
Added